source/components/defining-a-component.md

```diff
@@ -5,6 +5,10 @@ create a `components/blog-post` template.
 **Note:** Components must have at least one dash in their name. So `blog-post` is an acceptable name, so is `audio-player-controls`, but `post` is not. This prevents clashes with current or future HTML element names, and
 ensures Ember detects the components automatically.
 
+```bash
+ember generate component blog-post
+```
+
 A sample component template would look like this:
```

Comment:
I think @joostdevries should make this quick change so we can get it merged in - there are already several other places his addition of generators have been merged in, and if we go forward with #519 we can replace them all.